SwiftCompilerSources/Sources/SIL/Function.swift

```diff
@@ -108,6 +108,10 @@ final public class Function : CustomStringConvertible, HasShortDescription, Hash
     return nil
   }
 
+  public var declRef: BridgedDeclRef {
```

Comment:
Please don't expose bridged data structures in a public API.

Note that there is a `SIL.DeclRef`.
Also, the declref of a function can be null. So this should actually return an optional `DeclRef`. We should bridge the `SILDeclRef.isNull()` case to a `DeclRef?` nil.

Anyway, do you actually need this API?